libs/core/schematics/ng-add/index.ts

```diff
@@ -2,13 +2,13 @@ import { Rule, SchematicContext, Tree, chain, SchematicsException } from '@angul
 import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
 import { getPackageVersionFromPackageJson, hasPackage } from '../utils/package-utils';
 import { addPackageJsonDependency, NodeDependency, NodeDependencyType } from '@schematics/angular/utility/dependencies';
-import { addImportToRootModule, getProjectFromWorkspace, hasModuleImport } from '../utils/ng-module-utils';
-import { getAppModulePath } from '@schematics/angular/utility/ng-ast-utils';
-import { getWorkspace } from '@schematics/angular/utility/config';
+import { hasModuleImport } from '../utils/ng-module-utils';
 import { WorkspaceSchema } from '@schematics/angular/utility/workspace-models';
 
 import { cdkVersion } from './versions';
 import { defaultFontStyle } from './styles';
+import { addModuleImportToModule, getProjectFromWorkspace, getProjectMainFile, getAppModulePath } from '@angular/cdk/schematics';
```

Comment:
How do you solve this CDK dependency. ? CDK needed be installed even before npm install kicks in. If you dont have CDK installed , it works?

Reply:
@fkolar  Good point, it throws error; `An unhandled exception occurred: Cannot find module '@angular/cdk/schematics'`. Anyway in `fundamental-ngx/core` package.json, CDK lib is added to peerDependency list, so it's mandatory. Should I take some methods straight from cdk then, instead of importing?

Reply:
The cdk needs to be there before peerDependency ;-) But I am testing something here locally on some other project (blank empty), where I tried to do what you did, but somehow it reads CDK from somewhere. and it works. still not sure how its possible. 

Maybe cuz I am doing npm link? hmm.